cmd/cc-log-parser/README.md

```diff
@@ -0,0 +1,47 @@
+# `cc-log-parser`
+
+`cc-log-parser` is a tool that combines log files from various Clear
+Containers components. It sorts the logfiles by timestamp and re-displays
+them, adding a time delta showing how much time has elapsed between each log
+entry.
+
+## Component logfiles
+
+The primary two logs that are passed to the tool are the [runtime global
+log](https://github.com/clearcontainers/runtime#debugging) and the [proxy
+log](https://github.com/clearcontainers/proxy#debugging).
+
+The [virtcontainers](https://github.com/containers/virtcontainers) logs are
+automatically added to the runtimes global log.
+
+The [agent](https://github.com/clearcontainers/agent) logs are encoded inside
+the proxies log. The tool automatically unpacks these and displays only the
+agent log for these messages (the proxy message that encapsulates an agent
+message is discarded).
+
+## Usage
+
+To merge together all logs:
+
+1. Enable global logging in the runtime
+1. Enable debug logging for the proxy.
+1. Clear the systemd journal (optional)
+   ```
+   $ sudo systemctl stop systemd-journald
+   $ sudo rm -f /var/log/journal/*/* /run/log/journal/*/*
+   $ sudo systemctl start systemd-journald   
+   ```
+1. Create a container
+1. Collect the logs
+    1. Save the proxy log (which also includes agent debug)
+       ```
+       $ sudo journalctl -q -o cat -lau cc-proxy | grep time= > proxy.log
+       ```
+    1. Save the runtime log
+       ```
+       $ sudo cp /var/lib/clear-containers/runtime/runtime.log ./runtime.log
+       ```
+1. Run the script
+```
```

Comment:
indent this code block

Reply:
Thanks for reviewing @iphutch - branch updated.